lib/deobfuscate.py

```diff
@@ -7,6 +10,20 @@
 DEFAULT_MAX_DEPTH = 150
 
 
+def parse_base64_payload(payload: str) -> str:
+    if -1 == payload.find(r"Command/Base64/"):
+        return payload
+
+    logger.debug(f"Base64 command has been identify")
+    payload_without_base64, base64_command = payload.split(r"Command/Base64/")
+    try:
+        payload = payload_without_base64 + base64.b64decode(base64_command).decode()
+    except (binascii.Error, UnicodeDecodeError):
+        logger.debug(f"Got incorrect base64 payload - {base64_command}")
+    finally:
```

Comment:
No need for finally if that's the end of the function

Reply:
Fixed